.github/workflows/on-demand-vrfv2plus-smoke-tests.yml

```diff
@@ -70,6 +70,9 @@ jobs:
     with:
       custom_test_list_json: ${{ needs.set-tests-to-run.outputs.test_list }}
       chainlink_version: ${{ inputs.chainlink_version }}
+      slack_notification_after_tests: always
+      slack_notification_after_tests_name: "VRF V2 Plus Smoke Tests with test config: ${{ inputs.test_config_override_path || 'default' }}"
+      slack_notification_after_tests_notify_user_id_on_failure: U03PJF3VDB8
```

Comment:
this should not be hardcoded, as VRF QA activities will be hand over to Dev Services team. We need it to be optional and provided by the user

Reply:
@iljapavlovs you want user_id to be optional? Should there be any default? If yes, what?